### DISCUSSION TITLE
Is there a way to manually set %PDF% ?

### DISCUSSION THREAD

**Discussion by mil-ad** [James-Yu/LaTeX-Workshop#3434] [Q&A]
I use latexmk via some Makefile recipes (i.e. I skip LaTeX-Workshop's recipes) and because I set the name for the output file via latexmk's `--jobname` the `%PDF%` doesn't expand to the correct path.

Is there a way to explicitly set it? Perhaps in the configs?

I'm trying to get synctex working with sioyek PDF viewer and the file path shows up in a few places.

**Comment by Saltsmart** [comment]
Here is my `settings.json` in `.vscode` folder, you may change the `--jobname` here and don't need the Makefile:
```
    "latex-workshop.latex.tools": [
        {
            "name": "xelatexmk",
            "command": "latexmk",
            "args": [
                "-xelatex",
                "%DOC%",
                "-jobname=[specified pdf name]",  // PDF name
                "-outdir=%WORKSPACE_FOLDER%/Output", // Folder to output PDF and synctex files
                "-synctex=1",
                "-interaction=nonstopmode",
                "-auxdir=%WORKSPACE_FOLDER%/Temp", // temporary files
                "-file-line-error"
            ],
            "env": {}
        }
    ],
```
